cpp/src/converters/COOtoCSR.cuh

```diff
@@ -143,12 +141,12 @@ gdf_error ConvertCOOtoCSR(T* sources, T* destinations, int64_t nnz, CSR_Result<T
     ALLOC_FREE_TRY(unique, stream);
```

Comment:
Will an error be thrown by ALLOC_FREE_TRY? If so, then this is fine.
Otherwise, we need to revisit its usage.

Reply:
Yes, this is a wrapper from `rmm_utils.h`. It will end up calling `cugraph::detail::throw_rmm_error(status, __FILE__, __LINE__);` when RMM returns a failure.